CHANGELOG.md

```diff
@@ -1 +1,7 @@
 # HEAD
+
+# 1.0.0
+
+## Bug Fixes
+
+-   **js:** Transpile package using Babel (#6).
```

Comment:
Suggestion: changes should be added under `# HEAD`, unless you're planning on publishing `#1.0.0` straight away as soon as this PR merges. If that is the case, and you intend to publish as the next step, then it would be appropriate to change `# HEAD` to `# 1.0.0`, rather than adding in both.